File: backend/components/metiers/administration_publique/fixtures/samples/sample_fixture.test.js
// sample_fixture.test.js - Test unitaire pour sample_fixture.js (helpers/core/samples)
const sampleFixture = require('./sample_fixture');
describe('helpers/core/samples/sample_fixture.js', () => {
  it('doit exposer un objet sample correct', () => {
    expect(sampleFixture.sample).toBeDefined();
    expect(sampleFixture.sample.name).toBe('Sample Helper');
    expect(sampleFixture.sample.type).toBe('helper');
    expect(sampleFixture.sample.status).toBe('active');
  });
});

File: backend/components/metiers/administration_publique/fixtures/samples/sample_fixture.test.py
"""
Test unitaire pour sample_fixture.py (helpers/core/samples)
"""
from .sample_fixture import sample_helper_fixture

def test_sample_helper_fixture():
    fixture = sample_helper_fixture()
    assert isinstance(fixture, dict)
    assert fixture["name"] == "Sample Helper"
    assert fixture["type"] == "helper"
    assert fixture["status"] == "active"
